helion/language/device_print.py

```diff
@@ -0,0 +1,85 @@
+"""Device print support for Helion kernels."""
+
+from __future__ import annotations
+
+import ast
+from typing import TYPE_CHECKING
+
+from torch.fx import has_side_effect
+
+from .. import exc
+from .._compiler.ast_extension import create
+from .._compiler.ast_extension import expr_from_string
+from . import _decorators
+
+if TYPE_CHECKING:
+    import torch
+
+    from .._compiler.inductor_lowering import CodegenState
+
+
+@has_side_effect
+@_decorators.api()
+def device_print(prefix: str, *values: torch.Tensor) -> None:
+    """
+    Print values from device code.
```

Comment:
Nit: I have found that Google style docblocks typically have the best support in IDEs e.g. : 
```Py
    """Print values from device code.
    
    Args:
        prefix: A string prefix for the print statement
        values: Tensor values to print
    """
 ```
    
Not sure if we have any consistency here or if we are doing the above because it plays the best w/ some doc build?

Reply:
yeah I looked around in the codebase and the Sphinx/reStructuredText style docstrings is still more common. Maybe later we can do a pass to update all docstrings, if Google-style docblocks are better for IDEs.

Reply:
Sounds good 👍